NvimView/Sources/NvimView/NvimView+RemoteOptions.swift

```diff
@@ -0,0 +1,129 @@
+/**
+ * Renee Koecher -  @shirk
+ * See LICENSE
+ */
+
+import Cocoa
+import MessagePack
+import RxSwift
+
+extension NvimView {
+  enum RemoteOption {
+    // list of currently handled remote options
+    case guifont(fontSpec: String)
+    case guifontWide(fontSpec: String)
+
+    static func fromValuePair(_ option: (key: MessagePackValue, value: MessagePackValue)) -> RemoteOption? {
+      switch option.key.stringValue ?? "" {
```

Comment:
Rather than using nil coalescing operator, I would `guard let` on `stringValue`s of `key` and `value`, but just my taste. :)

Reply:
I had to dust off my Swift and lift it from somewhere between 3 & 4 to 5.3 ^^;